faunadb-common/src/main/java/com/faunadb/common/Connection.java

```diff
@@ -22,7 +22,7 @@
 
 /**
  * The HTTP Connection adapter for FaunaDB clients.
- *
+ * <p>
```

Comment:
Humm... I think it's just IntelliJ formatting classes. I can disable auto-formatting for docs from now on, if you think that is a problem.